dash/development/component_generator.py

```diff
@@ -19,6 +19,15 @@
 from ._py_components_generation import generate_classes_files
 
 
+forbidden_props = [
```

Comment:
reserved_props?

Reply:
But they're not props, they're words/patterns that you're not allowed to use as props. I suppose I could call them `reserved_words` if you don't like `forbidden`.

Reply:
`reserved_words` is better, just feel it's more accurate with `reserved`. and forbidden might be harsh